docs/tutorial.qmd

```diff
@@ -0,0 +1,980 @@
+---
+title: "Using Flow"
+tbl-colwidths: [30,70]
+---
+
+## Core Concepts
+
+### Flow Type System
+
+Inspect Flow mirrors Inspect AI's object model with corresponding Flow types:
+
+-   **`FlowConfig`** — The top-level workflow definition that translates to an Inspect AI [Eval Set](https://inspect.aisi.org.uk/eval-sets.html). Contains a list of tasks and workflow settings.
+-   **`FlowTask`** — Configuration for a single evaluation task. Maps to Inspect AI [Task](https://inspect.aisi.org.uk/tasks.html) parameters.
+-   **`FlowModel`** — Model configuration including API settings and generation settings. Maps to Inspect AI [Model](https://inspect.aisi.org.uk/models.html).
+-   **`FlowGenerateConfig`** — Model generation parameters. Maps to Inspect AI [GenerateConfig](https://inspect.aisi.org.uk/models.html#generation-config).
+-   **`FlowSolver`** and **`FlowAgent`** — Solver and agent chain configuration. Map to Inspect AI [Solver](https://inspect.aisi.org.uk/solvers.html) and [Agent](https://inspect.aisi.org.uk/agents.html).
+-   **`FlowOptions`** — Runtime execution options like concurrency limits, error handling, and logging preferences. Maps to Inspect AI [eval_set()](https://inspect.aisi.org.uk/eval-sets.html) parameters.
+-   **`FlowDefaults`** — System for setting default values across tasks, models, solvers, and agents.
+
+### Flow Config Structure
+
+`FlowConfig` is the primary interface for defining Flow workflows. All Flow operations—including [parameter sweeps and matrix expansions](#parameter-sweeping)—ultimately produce a list of tasks that `FlowConfig` executes.
+
+**Required fields:**
+
+-   **`tasks`** — List of `FlowTask` objects defining the evaluations to run
```

Comment:
Technically this isn't required. I expect that once we support including other config files there will be value in creating configs that do not have tasks.